modules/installation/pages/pubcloud-requirements.adoc

```diff
@@ -27,14 +27,14 @@ It is important to perform hostname and IP configuration before you set up {prod
 * {productname} Server and Proxy instances need to run in a network configuration that provides you control over DNS entries, but cannot be accessed from the internet at large.
 * Within this network configuration DNS resolution must be provided: `hostname -f` must return the fully-qualified domain name (FQDN).
 * DNS resolution is also important for connecting clients.
-* DNS is dependent on the cloud framework you choose, refer to the cloud service provider documentation for detailed instructions.
+* DNS is dependent on the cloud framework you choose, refer to the cloud provider documentation for detailed instructions.
```

Comment:
What's wrong with `cloud service provider`? That's what the likes of AWS and Azure are called.

Reply:
I was trying to align the terms a little better, per @keichwa's comment.